lms/static/coffee/fixtures/calculator.html

```diff
@@ -11,8 +11,8 @@
                 <li class="hint-item" id="hint-moreinfo" tabindex="-1">
                   <p>
                     <span class="bold">For detailed information, see
-                      <a id="hint-link-first" href="http://edx-guide-for-students.readthedocs.org/en/latest/SFD_mathformatting.html">Entering Mathematical and Scientific Expressions</a>in the
-                      <a id="hint-link-second" href="http://edx-guide-for-students.readthedocs.org/en/latest/index.html">edX Guide for Students </a> format
+                      <a id="hint-link-first" href="http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_mathformatting.html#math-formatting">Entering Mathematical and Scientific Expressions</a> in the
```

Comment:
Note that this is a test fixture file so strictly speaking it doesn't need to be updated. However it doesn't hurt (as long as the tests still pass) and it makes it less confusing for folks to have the correct URL.